client/app/scripts/charts/edge.js

```diff
@@ -5,6 +5,28 @@ import classNames from 'classnames';
 import { enterEdge, leaveEdge } from '../actions/app-actions';
 import { encodeIdAttribute, decodeIdAttribute } from '../utils/dom-utils';
 
+function isStorageComponent(id) {
+  if (id === '<persistent_volume>' || id === '<storage_class>' || id === '<persistent_volume_claim>') {
+    return true;
+  }
+  return false;
+}
+
+// getAdjacencyClass takes id which contains information about edge as a topology
+// of parent and child node.
+// For example: id is of form "nodeA;<storage_class>---nodeB;<persistent_volume_claim>"
+function getAdjacencyClass(id) {
+  const topologyId = id.split('---');
+  const fromNode = topologyId[0].split(';');
+  const toNode = topologyId[1].split(';');
+  if (fromNode[1] !== undefined && toNode[1] !== undefined) {
```

Comment:
Just checking that this logic is correct:
- If we see an edge like this: `nodeA;<storage_class>---nodeB` we do _not_ want to show a line?

Otherwise I think we're good.

Reply:
@foot 
> If we see an edge like this: nodeA;<storage_class>---nodeB we do not want to show a line?

We're adding one more wrapper over normal line only for storage components. If it's not storage component then we're sending whatever normal line was already there. Here, if `nodeB` is Internet node then it doesn't have any topology name i.e. `<topology-name>` then we'll not add any wrapper around it i.e `link-none` for the wrapper.